QuizApp/QuizApp/UI/Common/Fonts.swift

```diff
@@ -0,0 +1,25 @@
+import UIKit
+
+extension UIFont {
+
+    static let heading1 = UIFont(name: "SourceSansPro-Bold", size: 32)!
+    static let heading2 = UIFont(name: "SourceSansPro-Bold", size: 28)!
+    static let heading3 = UIFont(name: "SourceSansPro-Bold", size: 24)!
+    static let heading4 = UIFont(name: "SourceSansPro-Bold", size: 20)!
+    static let heading5 = UIFont(name: "SourceSansPro-Bold", size: 18)!
+    static let heading6 = UIFont(name: "SourceSansPro-Bold", size: 16)!
+
+    static let subtitle1 = UIFont(name: "SourceSansPro-SemiBold", size: 18)!
+    static let subtitle2 = UIFont(name: "SourceSansPro-SemiBold", size: 16)!
+    static let subtitle3 = UIFont(name: "SourceSansPro-SemiBold", size: 14)!
+    static let subtitle4 = UIFont(name: "SourceSansPro-SemiBold", size: 12)!
+
+    static let body1 = UIFont(name: "SourceSansPro-Regular", size: 16)!
+    static let body2 = UIFont(name: "SourceSansPro-Regular", size: 12)!
+
+    static let result = UIFont(name: "SourceSansPro-Bold", size: 88)!
+
+    static let leaderboardIndex = UIFont(name: "SourceSansPro-Black", size: 20)!
```

Comment:
This is one example of a mistake :) Use SourceSansPro-Bold & 20, AKA heading4 ;)